assets/stylesheets/common/direct-message-creator.scss

```diff
@@ -0,0 +1,189 @@
+.direct-message-creator {
```

Comment:
A general comment (not meant to be a blocker for this PR at all, more of a mental note): we need to extract some of the chat styling into "modular elements", so that it can be more easily reused.

Reply:
Yep, I have already started splitting a lot of components in last weeks, but we certainly need to do much more

Reply:
Awesome, thanks!